cppcache/integration-test/ThinClientVersionedOps.hpp

```diff
@@ -135,7 +135,7 @@ DUNIT_TASK_DEFINITION(CLIENT1, StartClient1)
     // 0/*redundancy*/, true/*clientNotification*/,
     // -1/*subscriptionAckInterval*/,
     // 5/*connections*/, 60000/*loadConditioningInterval*/);
-    //auto regPtr0 = createRegionAndAttachPool(regNames[0],USE_ACK,
+    // auto regPtr0 = createRegionAndAttachPool(regNames[0],USE_ACK,
     // nullptr);
```

Comment:
Any reason we can't just delete this commented-out code?

Reply:
This looks like the formatter just cleaned up this comment. I'd rather stay away from it in this commit. I used `clang-tidy` to automate the fixes and `clang-format` to cleanup the formatting after.